aiida/tools/importexport/common/archive.py

```diff
@@ -102,14 +198,56 @@ def filepath(self):
         """
         return self._filepath
 
+    @property
+    def output_filepath(self):
+        """Return the filepath of where the archive should be repacked
+
+        :return: the archive output filepath
+        """
+        return self._output_filepath
+
+    @property
+    def overwrite(self):
```

Comment:
Since overwrite should no longer be a class attribute but rather just a write method argument, it no longer needs a setter.